File: backend/internal/server/handler/admin/response/venue.go
package response

import "time"

// Venue represents a detailed view of a venue for admins.
type Venue struct {
	ID          int       `json:"id"`
	Name        string    `json:"name"`
	Location    string    `json:"location"`
	Description string    `json:"description"`
	CreatedAt   time.Time `json:"created_at"`
}

File: frontend/src/features/admin/types/venue.ts
import { Venue as EntityVenue } from '@entities/venue';

export interface Venue {
  id: number;
  name: string;
  location?: string;
  description?: string;
  createdAt: string;
}

export const toVenue = (entity: EntityVenue): Venue => ({
  id: entity.id,
  name: entity.name,
  location: entity.location,
  description: entity.description,
  createdAt: entity.createdAt,
});
